docs/snippets.md

```diff
@@ -5,6 +5,9 @@ sidebar: guides
 hideHelpfulForm: true
 ---
 
+> **Want to contribute?**  
+> Follow [this guide](https://github.com/tilt-dev/tilt.build#contributing-snippets) to find out how to your own snippets!
```

Comment:
I think this will need to be
```suggestion
> Follow [this guide](https://github.com/tilt-dev/tilt.build/blob/master/contributing-snippets.md) to find out how to your own snippets!
```